### PR TITLE
fix: support additional states

### PR DESCRIPTION
**Pull request by tinker-michaelj**
**Description**:
 - Add some fields to support additional states used.

### CODE REVIEW

File: block/stream/output/state_changes.proto [hunk 1]
```diff
@@ -810,4 +854,4 @@ message QueuePushChange {
  * information beyond the state name common to all state changes.
  */
 message QueuePopChange {
-}
+}
```
Comment (jsync-swirlds) [inline block/stream/output/state_changes.proto:857]: A newline is required at EOF (isn't enforced yet due to minimal CI support).

File: block/stream/output/state_changes.proto [hunk 2]
```diff
@@ -635,7 +658,7 @@ message MapChangeKey {
         /**
          * A change to a virtual map keyed by File identifier.
          */
-        proto.FileID filed_id_key = 4;
+        proto.FileID file_id_key = 4;
```
Comment (jsync-swirlds) [inline block/stream/output/state_changes.proto:661]: Thanks for fixing this.

File: block/stream/output/state_changes.proto [hunk 3]
```diff
@@ -41,8 +41,10 @@ option java_multiple_files = true;
 import "google/protobuf/wrappers.proto";
 import "basic_types.proto";
 import "exchange_rate.proto";
+import "state/addressbook/node.proto";
 import "state/blockrecords/block_info.proto";
 import "state/blockrecords/running_hashes.proto";
+import "state/blockstream/block_stream_info.proto";
```
Comment (jsync-swirlds) [inline block/stream/output/state_changes.proto:47]: Is this in another PR?
I cannot find this file.

**Review by jsync-swirlds — COMMENTED** [review]
One trivial (but required) fix and a file I cannot find.

File: block/stream/output/state_changes.proto (hunk 1 above)
Comment (tinker-michaelj) [reply block/stream/output/state_changes.proto]: Ah yes, added.

File: block/stream/output/state_changes.proto (hunk 3 above)
Comment (tinker-michaelj) [reply block/stream/output/state_changes.proto:47]: Oops, didn't add! Should be there now.

**Review by jsync-swirlds — COMMENTED** [review]
Some comments for the newly added file.

**Pull request merged**